assets/javascripts/openqa.js

```diff
@@ -577,19 +577,39 @@ function renderComments(row) {
 }
 
 function renderHttpUrlAsLink(value) {
-  const span = document.createElement('span');
-  for (let match; (match = value.match(/https?:\/\/[^\s,]*/)); ) {
-    const url = match[0];
-    const link = document.createElement('a');
-    link.href = url;
-    link.target = 'blank';
-    link.appendChild(document.createTextNode(url));
-    span.appendChild(document.createTextNode(value.substr(0, match.index)));
-    span.appendChild(link);
-    value = value.substr(match.index + url.length);
+  if (!value) {
+    return document.createTextNode('');
   }
-  span.appendChild(document.createTextNode(value));
-  return span;
+  if (Array.isArray(value)) {
+    const fragment = document.createDocumentFragment();
```

Comment:
it seems you can move this outside the if and then you can delete https://github.com/os-autoinst/openQA/pull/5972/files#diff-fae8284b51d426d67a2d9bb8590c1f8e06151d909a34a7e67f36bdc2038907e9R599 as well.

Reply:
yes, good catch :+1: